src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+import Display from "./Display";
+import "./App.css";
+
+const App = () => (
```

Comment:
You don't need this file in this PR. The only component we should define in this PR is Display.